.dockerignore

```diff
@@ -15,11 +15,14 @@ pip-log.txt
 pip-delete-this-directory.txt
 
 # Application data
-/data
+/docker/data
```

Comment:
From https://docs.docker.com/engine/reference/builder/#/dockerignore-file:
> For the purposes of matching, the root of the context is considered to be both the working and the root directory. For example, the patterns /foo/bar and foo/bar both exclude a file or directory named bar in the foo subdirectory

This seems correct, but I think that `docker/data` would also be fine, and is consistent with the rest of the file. I don't have a strong preference either way.